README.md

```diff
@@ -40,97 +49,44 @@ exit
 
 ### Configuration
 
-Cromwell can be configured via two files passed to the `-Dconfig.file` and `--options`
-command line arguments. We provide templates for both files, which are suitable for
-running workflows on Google Cloud using
-[Google Cloud Life Sciences](https://cromwell.readthedocs.io/en/stable/tutorials/PipelinesApi101/)
-(previously known as PAPI - Pipelines API).
-
-First edit `cromwell.template.conf` to replace the following values in
-angle brackets:
+Cromwell can be configured via two files passed to the `-Dconfig.file` and `--options` command line arguments. We provide templates for both files, which are suitable for running workflows on Google Cloud using [Google Cloud Life Sciences](https://cromwell.readthedocs.io/en/stable/tutorials/PipelinesApi101/) (previously known as PAPI - Pipelines API).
 
-- `<project>`: the Google Cloud project ID (eg: project-name-12312).
-- `<bucket>`: a Google Cloud Storage bucket name to store executions.
-- `database.db.password`: Update the password in case you changed the default (empty) password for your local MySQL server.
+**Important**: Make sure to schedule your VM workers in a region that's colocated with your data buckets, to avoid incurring high network egress costs. Don't move large files like BAMs between continents: copying 1 TB of data from the US to Australia costs 190 USD. Adjust the `default-zones` attribute in the template if necessary.
 
-**Important**: Make sure to schedule your VM workers in a region that's colocated with
-your data buckets, to avoid incurring high network egress costs. Don't move large files
-like BAMs between continents: copying 1 TB of data from the US to Australia costs 190
-USD. Adjust the `zones` attribute in the template if necessary.
+To run Cromwell, first edit `cromwell.conf` to replace the following hardcoded values:
```

Comment:
Would be good to update this to take advantage of @illusional's analysis-runner + Cromwell integration.